src/library/Truncate/Truncate.js

```diff
@@ -0,0 +1,106 @@
+/* @flow */
+import React, { Component } from 'react';
+import { canUseDOM } from 'exenv';
+import debounce from 'lodash.debounce';
+import { ellipsis } from 'polished';
+import { createStyledComponent } from '../styles';
+import _Tooltip from '../Tooltip';
+
+type Props = {
+  /** Content of Truncate */
+  children?: React$Node,
+  /** Maximum width of the content */
+  maxWidth?: number | string
+};
+
+type State = {
+  tooltip: boolean
+};
+
+export const componentTheme = (baseTheme: Object) => ({
+  // Truncate_borderColor_focus: baseTheme.borderColor_theme_focus,
```

Comment:
Is this theme necessary❓

Reply:
Nope, we caught this yesterday too, just haven't pushed that change yet.